MBBSEmu.Tests/Integration/Echo_Tests.cs

```diff
@@ -0,0 +1,29 @@
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
```

Comment:
move to `MBBSEmu.Tests.Integration` namespace and import `MBBSEmu.Tests.ExportedModules.Majorbbs`

Reply:
Copy/paste for the win!